src/Psalm/IssueBuffer.php

```diff
@@ -341,50 +341,6 @@ public static function add(CodeIssue $e, bool $is_fixable = false): bool
         return true;
     }
 
-    private static function removeRecordedIssue(string $issue_type, int $file_offset): void
-    {
-        $recorded_issues = self::$recorded_issues[self::$recording_level];
-        $filtered_issues = [];
-
-        foreach ($recorded_issues as $issue) {
-            [$from] = $issue->code_location->getSelectionBounds();
-
-            if ($issue::getIssueType() !== $issue_type || $from !== $file_offset) {
-                $filtered_issues[] = $issue;
-            }
-        }
-
-        self::$recorded_issues[self::$recording_level] = $filtered_issues;
-    }
-
-    /**
-     * This will try to remove an issue that has been added for emission
-     */
-    public static function remove(string $file_path, string $issue_type, int $file_offset): void
```

Comment:
Public functions of non-internal classes have to be marked `@deprecated` and can only be removed in a major release.